main.go

```diff
@@ -167,21 +170,36 @@ func main() {
 //   3. Run data conversion
 //   4. Generate report
 func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *ioStreams, outputFilePrefix string, now time.Time) error {
-	conv, err := schemaConv(driver, ioHelper)
-	if err != nil {
-		return err
-	}
-	// close the seekable file
+	conv := internal.MakeConv()
```

Comment:
Can we declare a `conv` variable here instead of making an actual instance? E.g., 

```go
var conv *internal.Conv
if !dataOnly {
    conv, err = schemaConv(driver, ioHelper)
} else {
    conv = internal.MakeConv()
}
```

Reply:
Yeah sure. That would be a better way. Done.